package.json

```diff
@@ -1,9 +1,12 @@
 {
     "name": "community-ui",
-    "version": "0.0.3",
+    "version": "0.0.0-development",
```

Comment:
Can you update this to 0.0.3-development, so that there will not be any merge conflict?